### PR TITLE
feat: Instructor detail card

### PR DESCRIPTION
**Pull request by 01001110J**
# Description

This PR adds the instructor details card to class detail view.

This PR resolves [PADV-1031](https://agile-jira.pearson.com/browse/PADV-1031)

## Change log
- New component `src/features/Classes/InstructorCard/index.jsx`
- Added the default inscructor image `src/assets/avatar.svg`
- Added the redirect button in `src/features/Classes/Class/ClassPage/index.jsx`.
- Added config to import assets in `webpack.dev.config.js` and `webpack.prod.config.js`

### Before
be

### After
af

af2

### How to test

Clone the Institution Portal MFE
```Bash
     git clone <url>
```
Install the packages 📦.
```Bash
     npm i
```
Start project.
```Bash
     npm start
```

Finally, go to http://localhost:1980/courses/:courseId/:classId and see the result.

### CODE REVIEW

File: src/features/Classes/InstructorCard/index.jsx
Comment (AuraAlba) [inline src/features/Classes/InstructorCard/index.jsx:1]: Class information is not loaded correctly when the class is not in the first page of the previous page

Comment (AuraAlba) [inline src/features/Classes/InstructorCard/index.jsx:1]: Class information is not loaded correctly when the class is not in the first page of the previous page

File: src/features/Classes/InstructorCard/index.jsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  );
+
+  const totalEnrolled = (classInfo?.numberOfStudents || 0)
+    + (classInfo?.numberOfPendingStudents || 0);
+
+  const extraInstructors = classInfo?.instructors?.length === 1
+    ? (classInfo?.instructors?.length || 0)
+    : (classInfo?.instructors?.length || 0) - 1;
+
+  const formatDateRange = (startDate, endDate) => {
```
Comment (AuraAlba) [inline src/features/Classes/InstructorCard/index.jsx:29]: Please create a utils file and move this function there so can be used in others components

File: src/features/Classes/InstructorCard/index.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+          <b className="mr-1">Enrollment:</b>
+          {classInfo?.minStudentsAllowed && (
+            <>minimum {classInfo.minStudentsAllowed}, </>
+          )}
+          enrolled {totalEnrolled}, maximum {classInfo?.maxStudents}
+        </div>
+      </div>
+      <div className="separator" />
+      <div className="instructor-details">
+        <h4 className="text-color text-uppercase mb-3">Instructor</h4>
```
Comment (AuraAlba) [inline src/features/Classes/InstructorCard/index.jsx:89]: Please check font-size in figma. This looks bigger than is intended to be

**Review by AuraAlba — CHANGES_REQUESTED** [review]

File: src/features/Classes/InstructorCard/index.jsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  const classes = useSelector((state) => state.classes.table.data);
+
+  const queryParams = new URLSearchParams(location.search);
+  const classIdQuery = queryParams.get('classId')?.replaceAll(' ', '+');
+
+  const [classInfo] = classes.filter(
+    (classElement) => classElement.classId === classIdQuery,
+  );
+
+  const totalEnrolled = (classInfo?.numberOfStudents || 0)
```
Comment (AuraAlba) [inline src/features/Classes/InstructorCard/index.jsx:29]: Enrolled number is not updated when I invite a student

File: src/features/Classes/data/slice.js [hunk 4]
```diff
@@ -37,6 +37,15 @@ export const classesSlice = createSlice({
     updateFilters: (state, { payload }) => {
       state.filters = payload;
     },
+    fillClassesTable: (state, { payload }) => {
+      state.table.data = payload;
```
Comment (AuraAlba) [inline src/features/Classes/data/slice.js:41]: If this is not information for a table as far I can see, please store it in another part

**Review by AuraAlba — CHANGES_REQUESTED** [review]

**Review by anfbermudezme — CHANGES_REQUESTED** [review]

**Review by AuraAlba — APPROVED** [review]

**Review by anfbermudezme — APPROVED** [review]

**Pull request merged**